packages/story-editor/src/app/history/karma/history.karma.js

```diff
@@ -25,6 +25,7 @@ describe('CUJ: Creator can View and Modify Document Settings: Navigating without
   beforeEach(async () => {
     fixture = new Fixture();
     await fixture.render();
+    await fixture.collapseHelpCenter();
```

Comment:
@BrittanyIRL I remember it was under discussion previously as well, but can't recall right now exactly what was the conclusion -- thoughts on just closing it by setting the local storage? I know it's not the correct scenario for when the user opens the editor the first time but one could argue that most of the times, the user would actually use the editor with the Help Center already closed, and it happens less often that the user uses the editor for the first time with the Help Center opened, so this could perhaps be tested separately instead by resetting the local storage for those specific tests.

Thoughts?

Reply:
yes, so this was what i tried initially when we ended up with adding it to the fixture! 

The problem is that when you first load the editor (which we treat every time as the first time for per test for consistency) that the help center will show up because that local storage isn't there. Then, if we update the value to have isOpen be false then we'd need to _reload_ the entire thing to get the help center to close. Whereas with this proposed solution, it's just a click that's waited for before going on our merry way to run those sweet sweet tests.